packages/common/mui/src/lib/bar-chart/bar-chart-tool-tip/bar-chart-tool-tip.tsx

```diff
@@ -0,0 +1,89 @@
+import { Box, styled } from '@mui/material';
+import { H5 } from '../../typography/typography';
+import { TooltipProps } from 'recharts';
+
+export interface BarChartToolTipDataProps {
+  dataPointOne?: {
+    lineColor: string;
+    lineType: 'dashed' | 'solid' | 'none';
+    title: (data: number) => string | JSX.Element;
+  };
+  dataPointTwo?: {
+    lineColor: string;
+    lineType: 'dashed' | 'solid' | 'none';
+    title: (data: number) => string | JSX.Element;
+  };
+  dataPointThree?: {
+    lineColor: string;
+    lineType: 'dashed' | 'solid' | 'none';
+    title: (data: number) => string | JSX.Element;
```

Comment:
can we rename the `title` prop to something that shows it is a function, like `formatTitle`?